kahuna/public/js/components/gr-permissions-filter/gr-permissions-filter.tsx

```diff
@@ -74,6 +84,7 @@ const PermissionsFilter: React.FC<PermissionsWrapperProps> = ({ props }) => {
   const [isChargeable, setIsChargeable] = useState(props.chargeable);
   const [selectedOption, setSelection] = useState(defPerms);
   const [currentIndex, setCurrentIndex] = useState(-1);
+  const [logoClick, setLogoClick] = useState<LogoClickProperties>({logoClick: false, showPaid:false});
```

Comment:
```suggestion
```

Reply:
Will test suggestion and confirm that it meets requirements - Thanks